kpatch-build/kpatch-elf.c

```diff
@@ -121,6 +121,20 @@ struct symbol *find_symbol_by_name(struct list_head *list, const char *name)
 	return NULL;
 }
 
+struct symbol *find_symbol_by_offset(struct list_head *list, struct section *sec,
+				     unsigned int offset)
+{
+	struct symbol *sym;
+
+	list_for_each_entry(sym, list, list)
+		if (sym->type == STT_FUNC &&
```

Comment:
Based on this check this would be `find_func_by_offset()`

Reply:
Yup